CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # MicroProfile Language Server Changelog
 
+## [0.8.0](https://github.com/eclipse/lsp4mp/milestone/9?closed=1) (June 15, 2023)
+
+### Enhancements
+ * Improve completion performance in properties files (resolve support, and item defaults). See [#389](https://github.com/eclipse/lsp4mp/issues/389).
+ * Warning for methods annotated with `@Query`/`@Mutation` whose class is not annotated correctly. See [#355](https://github.com/eclipse/lsp4mp/issues/355).
```

Comment:
(nitpick) The warning appears on the class name, not on the methods

Reply:
Changed to "Warning for type declarations that are incorrectly annotated while their methods are annotated with `@Query`/`@Mutation`."